container_port.go

```diff
@@ -1,9 +1,10 @@
 package libyaml
 
 type ContainerPort struct {
-	PrivatePort string `yaml:"private_port" json:"private_port" validate:"required"`
-	PublicPort  string `yaml:"public_port,omitempty" json:"public_port,omitempty" validate:"required_minapiversion=2.8.0"`
-	Interface   string `yaml:"interface,omitempty" json:"interface,omitempty"`
-	PortType    string `yaml:"port_type,omitempty" json:"port_type,omitempty"`
-	When        string `yaml:"when,omitempty" json:"when,omitempty"`
+	PrivatePort       string `yaml:"private_port" json:"private_port" validate:"required"`
+	PublicPort        string `yaml:"public_port,omitempty" json:"public_port,omitempty" validate:"required_minapiversion=2.8.0"`
+	PublicPortInitial string `yaml:"public_port_initial,omitempty" json:"public_port_initial,omitempty" validate:"required_minapiversion=2.8.0"`
```

Comment:
shouldn't this be higher than 2.8.0?

Reply:
or is that just the API version that added reparsing yaml upon upgrades?

Reply:
honestly it doesnt matter